src/py/mat3ra/notebooks_utils/ipython/entity/material/visualize.py

```
@@ -0,0 +1,184 @@
+from mat3ra.made.tools.convert import to_ase
+from mat3ra.utils.array import convert_to_array_if_not
+
+from ...ui import MaterialViewProperties, create_responsive_image_grid, get_viewer_html, get_viewer_js
+
+
+class ViewersEnum(str, Enum):
+    wave = "wave"
+    ase = "ase"
+
+
+def get_material_image(material: Material, title: str, rotation="0x,0y,0z", repetitions=[1, 1, 1]):
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)